docs/96_faq/index.md

```diff
@@ -73,6 +73,24 @@ to update the key pairs associated with an account permission. This command repl
 
 If you are using a third party wallet with a UI, they may provide the ability to update your account public and private key pairs.
 
+### What is a digital wallet
+
+A digital wallet, also called simply "wallet", is where the keys are stored. A wallet can also allow you to:
+
+* generate new keys,
```

Comment:
For bullets, we want to use the following rules for consistency and Style Guide adherence: 

1. Avoid starting a bullet point with a lowercase letter
2. Avoid using comma or semicolon after a bullet point

So we can change the bullets to:

* Generate new keys
* Delete keys from wallet
* Send, receive, and spend cryptocurrencies

Note: Starting a bullet point with a lower case letter is not incorrect. It's just a different style. For dev docs, we use the above suggested style.